docs/env_variables.md

```diff
@@ -517,6 +517,10 @@ checks or alter some of the more exotic semantics of the tool:
     (empty/non present) will add no tags to the metrics. For more information,
     see [rpc_statsd.md](rpc_statsd.md).
 
+  - `AFL_SYNC_TIME` allows you to specify a different minimal time (in minutes)
+    between fuzzing instances synchronization. Default sync time is 30 minutes,
+    note that time is halfed for -M main nodes.
```

Comment:
Doesn't the extra halving make it more complex than needed? Maybe only do it (once) if the time is not set?

Reply:
I thought it should have same behaviour like when sync_time is set in config.h. So you think that when user specify sync_time manually then we should omit implicit logic with halving value and use same times? In that case, I suggest to replace the environment variable with command line option. Then it could be possible to manually set proper sync times for both kinds of instances. The default algorithm with halving time remains if there is no option provided.

Reply:
You can set env variables for each node as well. I don't think it's important enough to justify a parameter.
In any case I'm not fighting to change the current behavior if you are happy with it, just pointing out it may be surprising

Reply:
Nice